packages/salesforcedx-vscode-lwc/src/testSupport/testRunner/testRunner.ts

```diff
@@ -95,7 +95,7 @@ export class TestRunner {
       runTestsByPathArgs = [];
     }
     const testNamePatternArgs = testName
-      ? ['--testNamePattern', `"${escapeStrForRegex(testName)}"`]
+      ? ['--testNamePattern', `${escapeStrForRegex(testName)}`]
```

Comment:
The variable `testName` is being reused here.  There's one instance declared on75, and another one here (line 97) line.  Though this is in the original code, I think it should be cleaned up and a different name be used.